src/ansys/fluent/core/utils/data_transfer.py

```diff
@@ -56,7 +58,9 @@ def transfer_case(
     """
     for idx in range(num_files_to_try):
         file_name = (file_name_stem or "temp_case_file_") + "_" + str(idx)
-        folder = os.getenv("TMP", os.getenv("TMPDIR", "."))
+        folder = tempfile.mkdtemp(
+            suffix=".txt", prefix="serverinfo-", dir=pyfluent.EXAMPLES_PATH
```

Comment:
You can use a different name for the folder

Reply:
Thank you for pointing this out.

Reply:
Done.